Tests/Resource/ResourceTests.swift

```diff
@@ -24,48 +24,35 @@ extension Todo: ContentMappable {
 extension Todo: StructuredDataRepresentable {
     public var structuredData: StructuredData {
         return [
-            "id": id.map({StructuredData.from($0)}) ?? nil,
-            "title": StructuredData.from(title),
-            "done": StructuredData.from(done)
+            "id": id.map({StructuredData($0)}) ?? nil,
+            "title": StructuredData(title),
+            "done": StructuredData(done)
```

Comment:
Not anymore.. I removed `from`, from `StructuredData` in favor of `StructuredData(variable)`. That could be rewritten as:

```swift
return [
    "id": id.map({.string($0)}) ?? .null,
    "title": .string(title),
    "done": .bool(done)
]
```